salt/modules/pagerduty_util.py

```diff
@@ -0,0 +1,390 @@
+# -*- coding: utf-8 -*-
```

Comment:
This is a separate file from the existing module only to ease development.  Could easily be combined, or (eventually) moved to util, once util is overridable.